static/monorail.yml

```diff
@@ -1432,7 +1437,7 @@ paths:
         - name: q
           in: query
           description: query object to pass through to waterline.
-          required: false
+          required: true
```

Comment:
i originally tried to make this work with Q set to FALSE but just couldn't. as soon as i switched it to TRUE, then all of sudden the queries would work. Let me try testing again by throwing in `nil`. https://github.com/emccode/docker-machine-rackhd/blob/master/rackhd.go#L155